tos.html

```diff
@@ -1,4 +1,4 @@
-<p>These Terms of Service (&quot;Terms&quot;) govern your use of any Codidact community hosted at <a href="https://codidact.com/">https://codidact.com/</a> or any subdomain thereof (the &quot;Service&quot;). These Terms apply to all uses of the Service, whether you sign up for an account or not. Your use of the Service constitutes your acceptance of these Terms.</p>
+<p>These Terms of Service (&quot;Terms&quot;) govern your use of any Codidact community hosted at <a href="https://codidact.com/">https://codidact.com/</a>, <a href="https://codidact.org/">https://codidact.org/</a> or any subdomain thereof (the &quot;Service&quot;). They are an agreement between you and Codidact C.I.C. ("We" or "Us"). These Terms apply to all uses of the Service, whether you sign up for an account or not. Your use of the Service constitutes your acceptance of these Terms.</p>
```

Comment:
For the first clause, I suggest the wording I proposed at https://github.com/codidact/docs/pull/52/files#diff-d433de41697e39d52e81cd8a50b1094fe2924f2ea352e1ea887c00929be0c8b2R1, which covers any future domains we might use.

(Please consider other wordsmithing changes on that PR; maybe we want to pull all that together here?)

Good call on saying explicitly that these terms are an agreement w/ the foundation!

Reply:
Sounds good to me. Shall I merge #52 into this PR so we can do the lot together?

Reply:
I think that's a good idea.  This line might cause a merge conflict, so if it's easier to manually apply the other changes and then close that PR, that's fine too.